API/Routes/MemoryUpdateRouter.js

```diff
@@ -0,0 +1,9 @@
+import MemoryUpdateController from '../Controllers/MemoryUpdateController';
+import express from 'express';
+
+
+const MemoryUpdateRouter = express.Router();
+
+MemoryUpdateRouter.post('/api/v1/memories/:id', MemoryUpdateController.updateMemory);
```

Comment:
you use put when making updates
```suggestion
MemoryUpdateRouter.put('/api/v1/memories/:id', MemoryUpdateController.updateMemory);
```